pkg/fwrequest/fwrequest.go

```diff
@@ -120,8 +120,8 @@ func NewEntityFwRequest(
 	}
 	if !id.IsEmpty() {
 		r.RegisteredWith(id)
-		r.populateCommonAttributes(intMeta, agentVersion)
 	}
+	r.populateCommonAttributes(intMeta, agentVersion)
```

Comment:
Populates all metrics with integration metadata, no matter if they are registered or not.